src/carousel.js

```diff
@@ -726,6 +728,7 @@ const Carousel = createReactClass({
   },
 
   onResize() {
+    this.props.onResize();
```

Comment:
Shouldn't this fire _after_ the resize is done?

Reply:
+1

Reply:
+1